src/logics/stargate-v2/configs.ts

```diff
@@ -0,0 +1,433 @@
+      },
+      {
+        id: PoolId.ETH,
+        token: common.metisTokens.WETH,
+        address: '0x36ed193dc7160D3858EC250e69D12B03Ca087D08',
+        destinations: [
+          { chainId: common.ChainId.mainnet, endpointId: EndpointId.ETHEREUM },
+          { chainId: common.ChainId.optimism, endpointId: EndpointId.OPTIMISM },
+          { chainId: common.ChainId.base, endpointId: EndpointId.BASE },
+          { chainId: common.ChainId.iota, endpointId: EndpointId.IOTA },
+          { chainId: common.ChainId.arbitrum, endpointId: EndpointId.ARBITRUM },
+        ],
+      },
+      {
+        id: PoolId.USDT,
+        token: common.metisTokens['m.USDT'],
+        address: '0x4dCBFC0249e8d5032F89D6461218a9D2eFff5125',
+        destinations: [
+          { chainId: common.ChainId.mainnet, endpointId: EndpointId.ETHEREUM },
+          { chainId: common.ChainId.optimism, endpointId: EndpointId.OPTIMISM },
+          { chainId: common.ChainId.bnb, endpointId: EndpointId.BNB },
+          { chainId: common.ChainId.polygon, endpointId: EndpointId.POLYGON },
+          { chainId: common.ChainId.iota, endpointId: EndpointId.IOTA },
+          { chainId: common.ChainId.arbitrum, endpointId: EndpointId.ARBITRUM },
+          { chainId: common.ChainId.avalanche, endpointId: EndpointId.AVALANCHE },
+        ],
+      },
+    ],
+  },
+  {
+    eid: EndpointId.BASE,
+    chainId: common.ChainId.base,
+    pools: [
+      {
+        id: PoolId.ETH,
+        token: common.baseTokens.ETH,
+        address: '0xdc181Bd607330aeeBEF6ea62e03e5e1Fb4B6F7C7',
+        destinations: [
+          { chainId: common.ChainId.mainnet, endpointId: EndpointId.ETHEREUM },
+          { chainId: common.ChainId.optimism, endpointId: EndpointId.OPTIMISM },
+          { chainId: common.ChainId.metis, endpointId: EndpointId.METIS },
+          { chainId: common.ChainId.iota, endpointId: EndpointId.IOTA },
+          { chainId: common.ChainId.arbitrum, endpointId: EndpointId.ARBITRUM },
+        ],
+      },
+      {
+        id: PoolId.USDC,
+        token: common.baseTokens.USDC,
+        address: '0x27a16dc786820B16E5c9028b75B99F6f604b5d26',
+        destinations: [
+          { chainId: common.ChainId.mainnet, endpointId: EndpointId.ETHEREUM },
+          { chainId: common.ChainId.optimism, endpointId: EndpointId.OPTIMISM },
+          { chainId: common.ChainId.polygon, endpointId: EndpointId.POLYGON },
+          { chainId: common.ChainId.iota, endpointId: EndpointId.IOTA },
+          { chainId: common.ChainId.arbitrum, endpointId: EndpointId.ARBITRUM },
+          { chainId: common.ChainId.avalanche, endpointId: EndpointId.AVALANCHE },
+        ],
+      },
+    ],
+  },
+  {
+    eid: EndpointId.IOTA,
+    chainId: common.ChainId.iota,
+    pools: [
+      {
+        id: PoolId.ETH,
+        token: common.iotaTokens.WETH,
+        address: '0xdc181Bd607330aeeBEF6ea62e03e5e1Fb4B6F7C7',
```

Comment:
```suggestion
        address: '0x9c2dc7377717603eB92b2655c5f2E7997a4945BD',
```

https://stargateprotocol.gitbook.io/stargate/v/v2-developer-docs/technical-reference/mainnet-contracts#iota

StargateOFTETH.sol